src/jsonValidator.js

```diff
@@ -0,0 +1,14 @@
+import { keyTypes } from './constants.js'
+import { createError } from './error.js';
+
+export const jsonValidator=(jsonString) => {
+  const err = [];
+
+  try {
+    const checkJson = JSON.parse(jsonString);
+  } catch (error) {
+    err.push(createError("invalid-json"));
```

Comment:
This is not used. Its created inside the function scope, and we don't return it.
What's the purpose of this?